uvls/src/core/ast/transform.rs

```diff
@@ -1174,12 +1174,13 @@ pub fn visit_root(source: Rope, tree: Tree, uri: Url, timestamp: Instant) -> Ast
         state.connect();
         (state.ast, state.errors)
     };
-    let mut path = uri_to_path(&uri).unwrap();
-    if let Some(ns) = ast.namespace.as_ref() {
-        let len = path.len().saturating_sub(ns.names.len());
-        path.truncate(len);
-        path.extend_from_slice(&ns.names);
-    }
+    let path = uri_to_path(&uri).unwrap();
+    // without this Code namespaces are ignored for imports and the normal path is used 
```

Comment:
I think it's a good idea to keep this within the code.  
Does this work like a "switch" or does it need additional changes to make the import work like before?